jsx/Form.js

```diff
@@ -30,12 +39,56 @@ var FormElement = React.createClass({
       id: null,
       method: 'POST',
       class: 'form-horizontal',
+      columns: 1,
       fileUpload: false,
       onSubmit: function() {
         console.warn('onSubmit() callback is not set!');
+      },
+      onUserInput: function() {
+        console.warn("onUserInput() callback is not set!");
       }
     };
   },
+  getFormElements: function() {
+    var formElementsHTML = [];
+    var columns = this.props.columns;
+    var maxColumnSize = 12;
+    var colSize = Math.floor(maxColumnSize / columns);
+    var colClass = "col-xs-12 col-sm-" + colSize + " col-md-" + colSize;
+
+    // Render elements from JSON
+    var filter = this.props.formElements;
+    var userInput = this.props.onUserInput;
+    if (filter) {
```

Comment:
If we have a default of `[]` for `formElements` we can remove this conditional and use `map`.

```javascript
const formElementsHTML = this.props.formElements.map((element) => {
  return (
    <div className={colClass}>
      <LorisElement element={element} onUserInput={userInput} />
    </div>
  );
});
```